apps/landingPage/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Margarita's landing page",
```

Comment:
Sorry, I should have mentioned in latest commit,  but can we keep naming convention. `@kiwicom/margarita-landing-page`

Reply:
Will fix now, and should i keep description blanked as it is now?